index.html

```diff
@@ -1,24 +1,65 @@
-            <li class="treat-yourself__card treat-yourself__card_type_maincure"><img class="treat-yourself__card-image" src="/images/manicure.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Manicure</p></div></div></li>
-            <li class="treat-yourself__card treat-yourself__card_type_hair"><img class="treat-yourself__card-image" src="/images/hair.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Hair</p></div></div></li>
-            <li class="treat-yourself__card treat-yourself__card_type_male-grooming"><img class="treat-yourself__card-image" src="/images/maleGrooming.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Male Grooming</p></div></div></li>
+            <li class="treat-yourself__card treat-yourself__card_type_massage"><img class="treat-yourself__card-image"
+                    src="/images/massage.png">
+                <div class="treat-yourself__card-text">
+                    <p class="treat-yourself__text">Massage</p>
+                </div>
+                </div>
+            </li>
+            <li class="treat-yourself__card treat-yourself__card_type_waxing"><img class="treat-yourself__card-image"
+                    src="/images/waxing.png">
+                <div class="treat-yourself__card-text">
+                    <p class="treat-yourself__text">Waxing</p>
+                </div>
+                </div>
+            </li>
+            <li class="treat-yourself__card treat-yourself__card_type_make-up"><img class="treat-yourself__card-image"
+                    src="/images/makeUp.png">
+                <div class="treat-yourself__card-text">
+                    <p class="treat-yourself__text">Make-Up</p>
+                </div>
+                </div>
+            </li>
+            <li class="treat-yourself__card treat-yourself__card_type_maincure"><img class="treat-yourself__card-image"
+                    src="/images/manicure.png">
+                <div class="treat-yourself__card-text">
+                    <p class="treat-yourself__text">Manicure</p>
+                </div>
+                </div>
+            </li>
+            <li class="treat-yourself__card treat-yourself__card_type_hair"><img class="treat-yourself__card-image"
+                    src="/images/hair.png">
+                <div class="treat-yourself__card-text">
+                    <p class="treat-yourself__text">Hair</p>
+                </div>
+                </div>
+            </li>
+            <li class="treat-yourself__card treat-yourself__card_type_male-grooming"><img
+                    class="treat-yourself__card-image" src="/images/maleGrooming.png">
+                <div class="treat-yourself__card-text">
+                    <p class="treat-yourself__text">Male Grooming</p>
+                </div>
+                </div>
+            </li>
         </ul>
     </section>
+    <div class="divider"></div>
```

Comment:
I'd actually suggest use of the `<hr>` tag. (Horizontal rule) specifically designed for making this kind of break, rather than using a div

Reply:
Today I learned... 
Love it! I'll change it and send it back up.